projects/auth0-angular/src/lib/auth.interceptor.ts

```diff
@@ -77,6 +77,15 @@ export class AuthHttpInterceptor implements HttpInterceptor {
         return true;
       }
 
+      // If the URL ends with an asterisk, match using startsWith.
+      if (
+        typeof value === 'string' &&
+        value.indexOf('*') === value.length - 1 &&
```

Comment:
`value.indexOf('*') === value.length - 1`

If I recall correctly, **indexOf** will return `-1` when it's not found. And **value.length** of an empty string is `0`. So this line will evaluate to true. The previous one is true as well, because it's still a string, even if empty. And the one below I think it will 💥 when it tries to **substring(0, -1)**. 

Can we add a test for this scenario?

Reply:
I've added an empty string to the config, as well as putting in a guard for this function that exits early when `value` is falsy.